### ISSUE TITLE
Add a trophy for any liquid that can be pumped infinitely via create

### ISSUE THREAD

**Issue by TahoeBennie** [EnigmaticaModpacks/Enigmatica6#4981]
### Describe your suggestion

I would like there to be a trophy for all of the liquids that can be pumped out infinitely with a hose pulley. We already have these for lava, water, and oil, and it is basically necessary to use create to pump out enough oil at a good speed, so I would like a similar trophy for the other possible liquids and everything on the list of avaliable liquids that can be done infinitely via create.

### Why would you like this added/changed?

Just to add something else to be collected since enough oil and lava to get the oil and lava trophies is typically gotten through create bottomless pumping. The liquids, such as latex and menril resin, and any others that can be pumped infinitely should get their own trophy to be collected after getting 2.147m buckets of it in the supreme black hole tank just like lava and oil (and water).

**Comment by NielsPilgaard** [comment]
Good idea, I'm not sure we'll have time to implement it though - we'll see 😄
